### ISSUE TITLE
Any chance this is something simple due to a home assistant update?

### ISSUE THREAD

**Issue by thegreatarm** [jslove/xap_controller#10]
I have recently migrated HA to a new intel NUC.  This integration has worked flawlessly for me for a while, but when I try to install it via HACS to my new installation it doesn't work.   Any chance for a code update?

Logger: homeassistant.components.hassio
Source: components/hassio/__init__.py:694
Integration: Home Assistant Supervisor ([documentation](https://www.home-assistant.io/integrations/hassio), [issues](https://github.com/home-assistant/home-assistant/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+hassio%22))
First occurred: 4:15:23 PM (1 occurrences)
Last logged: 4:15:23 PM

The system cannot restart because the configuration is not valid: Invalid config for [media_player.xap_controller]: required key not provided @ data['sources']. Got None. (See ?, line ?)

**Comment by jslove** [comment]
This an old issue I realize, but if its still happening and you want me to take a look please post your config entries.

**Comment by thegreatarm** [comment]
Thank you for reaching out.  I still have two of the xap controller and would absolutely love to get them working again.  I had abandoned the thought of ever getting them working.  I will hook them back up, reproduce the issue and send you the configs.  Hell, maybe it will work.  I am several iterations forward on Home Assistant since posting this originally.  Are you still running these controllers in your own configs?  Thanks again :)

**Comment by jslove** [comment]
Yes, I still use them. There was a recent change to some HA code that broke
it but I just fixed it.


Jay Love
***@***.***

**Comment by thegreatarm** [comment]
Great news!  I'll give it a try and let you know how it goes.  Thanks again for reaching out.  I thank you for your work :)

**Comment by thegreatarm** [comment]
My XAP800's are working in HA again.  Thanks for the update!
